habitat-lab/habitat/config/benchmark/rearrange/ovmm.yaml

```diff
@@ -68,5 +68,6 @@ habitat:
       enable_physics: True
     needs_markers: false
   dataset:
+    data_path: data/episodes/rearrange/v4/val/cat_npz-exp-with_init_poses.json.gz
```

Comment:
The episodes will also need to move to ovmm folder. Also make sure that you call the episode dataset version v1